catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/controllers/V2WorksController.scala

```diff
@@ -14,12 +18,13 @@ class V2WorksController @Inject()(
   apiConfig: ApiConfig,
   elasticConfig: ElasticConfig,
   worksService: WorksService)(implicit ec: ExecutionContext)
-    extends WorksController(
+    extends WorksController[V2MultipleResultsRequest, V2SingleWorkRequest](
       apiConfig = apiConfig,
       indexName = elasticConfig.indexV2name,
       worksService = worksService
     ) {
   implicit protected val swagger = ApiV2Swagger
+  lazy override protected val includeParameterName: String = "include"
```

Comment:
minor: Do you need the `override`? Since `WorksController` doesn't define a value, you're not overriding anything.

Reply:
In this case probably not, because as you say, WorksController doesn't give it a value so it wouldn't compile anyway. I tend as a habit to put overrides a bit everywhere though, cause it makes compilation fail in cases where there is a default value in the base class, typos etc. It makes a bit more robust in case of further modifications and refactorings I think.